Dockerfile

```diff
@@ -27,8 +27,7 @@ WORKDIR /app
 
 COPY --chown=topher:topher pyproject.toml uv.lock ./
 
-RUN --mount=type=cache,target=/root/.cache/pip \
-    --mount=type=cache,target=/root/.cache/uv \
+RUN --mount=type=cache,target=/home/topher/.cache/uv,uid=1000,gid=1000 \
```

Comment:
DevinAI should we use the GROUP and USER IDs instead of hardcoding the values here?